src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -38,6 +41,23 @@ public override JObject ToJson()
 
         public override bool Verify(DataCache snapshot, Transaction tx)
```

Comment:
`Verify` will return true for all txs who will be packed in one block.
Image that we send the [2,690,038](https://testnet.neotube.io/block/2690038) - [2,690,040](https://testnet.neotube.io/block/2690040) block's tx in one block instead of 3 blocks.

If you want to set a limit, please add the limit to mempool, too.

Reply:
@AnnaShaleva Could you propose a change for `MemoryPool.CheckConflicts`?

Reply:
Sure, I'll create a PR.

Reply:
It's in #2911.